modules/storage-persistent-storage-fsGroup.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+
+[id="using_fsGroup_{context}"]
+= Using fsGroup to reduce pod timeouts
+
+If a storage volume contains many files (~1,000,000 or greater), you may experience pod timeouts.
```

Comment:
Would be possible to eliminate the parenthetical if this were a specific technical limit, e.g. "If a storage volume contains more than X files..."